apps/meteor/client/views/marketplace/AppsPage/AppsFilters.tsx

```diff
@@ -45,27 +45,32 @@ const AppsFilters = ({
 	statusFilterOnSelected,
 }: AppsFiltersProps): ReactElement => {
 	const t = useTranslation();
+	const context = useRouteParameter('context');
```

Comment:
instead of using the route once more, why not receive this through parameters?

Reply:
Solved, also solved a margin bug.